src/EscrowVerification/UserDetailsForm.tsx

```diff
@@ -45,20 +47,26 @@ export const phoneToFormattedString = (phonenumber: string): string => {
   return `(${areaCode}) ${firstPart}-${secondPart}`
 }
 
+type Posix = number
+
 const UserDetailsForm: FunctionComponent = () => {
   useStatusBarEffect("dark-content", Colors.background.primaryLight)
   const { t } = useTranslation()
   const navigation = useNavigation()
   const { successFlashMessageOptions } = Affordances.useFlashMessageOptions()
 
   const { testDate, setTestDate } = useEscrowVerificationContext()
-
   const [phoneNumber, setPhoneNumber] = useState("")
   const [isLoading, setIsLoading] = useState(false)
-  const [isFocused, setIsFocused] = useState(false)
+  const [isPhoneInputFocused, setIsPhoneInputFocused] = useState(false)
   const [errorMessage, setErrorMessage] = useState(defaultErrorMessage)
+  const [showDatePickerAndroid, setShowDatePickerAndroid] = useState(false)
 
-  const handleOnChangeTestDate = (testDate: Date | undefined) => {
+  const handleOnChangeTestDate = (
+    _event: Event,
+    testDate: Date | undefined,
+  ) => {
+    setShowDatePickerAndroid(false)
```

Comment:
This line seems weird, but there's a really UI bug that occurs if it's not included. Found in the issues for the date picker library.

Reply:
Could we make sure to mention this in the commit message?

Reply:
and add the issue link here as well?

Reply:
https://github.com/react-native-datetimepicker/datetimepicker/issues/263
https://github.com/react-native-datetimepicker/datetimepicker/issues/54